src/mediacatch_s2t/uploader.py

```diff
@@ -55,15 +78,17 @@ def _transcript_link(self):
     @staticmethod
     def _ffprobe(file_path) -> FFProbeResult:
         command_array = ["ffprobe",
-                        "-v", "quiet",
-                        "-print_format", "json",
-                        "-show_format",
-                        "-show_streams",
-                        file_path]
-        result = subprocess.run(command_array, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+                         "-v", "quiet",
```

Comment:
This is a linter update. Nothing is changed in the code.
Automatically done by pycharm :grimacing: